0_portfolio_input_check_functions.R

```diff
@@ -995,14 +995,14 @@ get_and_clean_company_fin_data <- function() {
 
   # comp_fin_data_raw <- comp_fin_data_raw %>% select(
   #   company_id, company_name, bloomberg_id, country_of_domicile, corporate_bond_ticker, bics_sector, bics_subgroup,
-  #   icb_subgroup, mapped_sector, has_asset_level_data, has_assets_in_matched_sector, sectors_with_assets,
+  #   icb_subgroup, financial_sector, has_asset_level_data, has_assets_in_matched_sector, sectors_with_assets,
   #   current_shares_outstanding_all_classes, company_status, market_cap, bond_debt_out,
   #   financial_timestamp
   # )
```

Comment:
Nit: Are we following up with a PR to remove this dead code?

Reply:
while philosophically I agree with cleaning up and removing "dead" code, I'm not currently in a position to determine whether or not this code is truly "dead" or if it's commented out for a good reason

Reply:
I know, that's precisely why it's a bad idea to comment code out in the first place -- nobody wants to take responsability for deleting it. I say we delete it. If we need it, we can find it with something like

```bash
git log -S "comp_fin_data_raw"
```

Reply:
this very PR, for instance, has what I consider to be "legitimate" use of commented code...

https://github.com/2DegreesInvesting/PACTA_analysis/blob/ac4802218c57fab978e04d521376afe21ea2a05e/web_tool_script_3.R#L251-L259

those arguments for the stress testing stuff are removed from the current master of `create_iteractive_report` so this PR will fail/break if I leave them in, but they will very likely need to be added back in once the stress test stuff settles, and i.m.o. it's easier and safer to uncomment rather than manually re-adding them.

Reply:
Good, for every rule there is always a good reason to not follow it :)